broadinstitute/single_cell_portal_core
# Fix security issue in activestorage (SCP-4192)

This fixes a security vulnerability in [`activestorage` reported by Dependabot](https://github.com/broadinstitute/single_cell_portal_core/security/dependabot/28).

To test:
* Terminate `rails` and `vite`
* Run `bundle update rails`
* Run `bundle list | grep activestorage`
* Confirm output includes "* activestorage (6.1.4.7)"
* Start Rails server and Vite
* Confirm local SCP loads

It satisfies SCP-4192.
